sig-cloud-provider/annual-report-2021.md

```diff
@@ -23,62 +32,59 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
+2. What metrics/community health stats does your group care about and/or
+   measure?
 
-   -
-   -
-   -
+   - We don't currently measure any statistics regarding community health.
 
-3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
-   to activities or programs that provide useful context or allow easy participation?
+3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group
+specifically by pointing to activities or programs that provide useful context
+or allow easy participation?
 
-   -
+   - Adding a new CONTRIBUTING.md in the same PR as this report.
 
-4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
-   does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
+4. If your group has special training, requirements for reviewers/approvers, or
+processes beyond the general [contributor guide], does your [CONTRIBUTING.md]
+document those to help **existing** contributors grow throughout the
+[contributor ladder]?
 
-   -
+   - N/A (we don't have special requirements beyond the general [contributor
+     guide].
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, we have contributors from Amazon, Google, VMWare, and others.
 
-6. Are there ways end users/companies can contribute that they currently are not?
-   If one of those ways is more full time support, what would they work on and why?
+6. Are there ways end users/companies can contribute that they currently are
+not?  If one of those ways is more full time support, what would they work on
+and why?
 
-   -
-   -
+   - Yes, we could use help on the extraction/migration effort, including
+     migrating tests out of the core Kubernetes repository and into each
+     cloud-provider repository.
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-
-Include any other ways you measure group membership
+- Primary slack channel member count: 942
+- Primary mailing list member count: 240
+- Primary meeting attendee count (estimated, if needed): 11
+- Primary meeting participant count (estimated, if needed): 11
+- Unique reviewers for SIG-owned packages: TBD, we have a lot of SIG-owned packages
+- Unique approvers for SIG-owned packages: TBD, we have a lot of SIG-owned packages
```

Comment:
You should be able to derive some of this with https://github.com/kubernetes-sigs/maintainers.

Reply:
Nice.  I ended up collecting them here but its not very dynamic (see this [gist](https://gist.github.com/nckturner/cddd64bc1a56eaec836c07a24f7fecf4).  Also may have missed repos and/or nested owner files.